README.md

```diff
@@ -1,2 +1,45 @@
 # terraform-aws-sensor
+
 Terraform for Corelight's AWS Cloud Sensor Deployment.
+
+<img src="docs/overview.png" alt="overview">
+
+## Usage
+```terraform
+
+module "sensor" {
+  source = "github.com/corelight/terraform-aws-sensor"
+
+  auto_scaling_availability_zones = ["<first az>", "<second az>"]
+  aws_key_pair_name = "<key pair name>"
+  
+  # Request access to Corelight sensor AMI from you Account Executive
+  corelight_sensor_ami_id = "<sensor AMI ID>"
+  license_key = "<your Corelight sensor license key>"
+  management_subnet_id = "<management subnet>"
+  monitoring_subnet_id = "<monitoring subnet>"
+  sensor_api_password = "<password for the sensor api>"
```

Comment:
can we name this `community_string` ? Will make a bit more sense for those that end up using fleet when that time comes.